app-modules/service-management/src/Filament/Resources/IncidentResource/Pages/ViewIncident.php

```diff
@@ -67,6 +67,12 @@ public function infolist(Infolist $infolist): Infolist
                             ->label('Status'),
                         TextEntry::make('assignedTeam.name')
                             ->label('Assigned Team'),
+                        TextEntry::make('created_at')
+                            ->datetime(config('project.datetime_format') ?? 'Y-m-d H:i:s')
+                            ->label('Created'),
+                        TextEntry::make('updated_at')
+                            ->datetime(config('project.datetime_format') ?? 'Y-m-d H:i:s')
```

Comment:
Let's just stick with the `'Y-m-d H:i:s'` for the datatime formatting configuration. `config('project.datetime_format')` is something legacy we will probably remove in the future anyways.